config/production/airflow-vpc.yaml

```diff
@@ -0,0 +1,10 @@
+template:
+  type: http
+  url: "{{stack_group_config.aws_infra_templates_root_url}}/v0.3.0/templates/VPC/vpc-20-private.yaml"
+stack_name: airflow-vpc-test
+
+parameters:
+  VpcName: airflow-vpc-test
+  VpcSubnetPrefix: "172.22"
```

Comment:
This value needs to be unique in order for private IP addresses to be resolved unambiguously. I've reserved the `172.24.0.0/20` CIDR block for this VPC ([source](https://sagebionetworks.jira.com/wiki/spaces/IT/pages/352976898/Infrastructure+Inventory#:~:text=orca%2Dvpc%20(TBD)-,172.24.0.0/20,-VPC%20for%20Airflow)).

Reply:
changed in latest commit